### PR TITLE
Bootloader 2.1.9

### PR DESCRIPTION
**Pull request by matejcik**
chore commits for #4195

### CODE REVIEW

File: core/embed/bootloader/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 2.1.9 [September 2024]
+
+### Changed
+
+- [T3T1] Allow starting bootloader without touch controller.  [#4195]
```
Comment (TychoVrahe) [inline core/embed/bootloader/CHANGELOG.md:11]: Not entirely sure about this, its only allowed in manufacturing mode, it shouldn't affect regular users.

Comment (matejcik) [reply core/embed/bootloader/CHANGELOG.md:11]: hmmm ok so how about something like this?

> Ignore touch errors while in manufacturing.

Comment (TychoVrahe) [reply core/embed/bootloader/CHANGELOG.md:11]: i think its better

Comment (matejcik) [reply core/embed/bootloader/CHANGELOG.md]: force-pushed

**Review by TychoVrahe — APPROVED** [review]

**Pull request merged**